lib/defaults/defaults.go

```diff
@@ -610,6 +610,8 @@ const (
 	KubeSystemNamespace = "kube-system"
 	// MonitoringNamespace is the name of k8s namespace for the monitoring-related resources
 	MonitoringNamespace = "monitoring"
+	// OpenEBSNamespace is the name of k8s namespace where OpenEBS is deployed.
+	OpenEBSNamespace = "openebs"
```

Comment:
Should we tie openebs to an "openebs" namespace, or simply tie it to a "storage" namespace, that's used by the "storage" app. Not a huge deal or anything, but might be slightly more consistent to match the two. Either openebs app -> openebs ns, or storage -> storage.

Thoughts?

Reply:
That's an interesting thought, I'm not sure we should necessarily tie those together though. The way I think about it:

* We could change namespace to a generic `storage` but that would complicate upgrading a bit as we'd have to diligently change all namespaces in their operator.yaml. Probably wouldn't be such a big deal if/when we migrate to Helm though.
* If we ever add another storage provider, it'd make sense to put it in a separate namespace. For the same reason, I think renaming the app to specifically "openebs" is also not ideal.

On the other hand, we do put all prometheus/alertmanager stuff into "monitoring" namespace... A nice side effect with using `openebs` namespace is that this is their default namespace so all command in their documentation just work.